5.js

```diff
@@ -0,0 +1,19 @@
+const disemvowel = string => arrayToString(filterOutSpaces(filterOutVowels(stringToArray(string))))
+
+const stringToArray = string => string.split('')
+
+const arrayToString = array => array.join('')
+
+const filterOutVowels = characters => characters.filter(
+  character => character !== 'a'
+            && character !== 'a'
+            && character !== 'i'
+            && character !== 'o'
+            && character !== 'u'
+)
+
+const filterOutSpaces = characters => characters.filter(
+  character => character !== ' '
+)
+
+module.exports = disemvowel
```

Comment:
@DianaVashti Can you give an example? It doesn't seem possible to me since filter and reduce are array methods.